config/config.json.template

```diff
@@ -40,6 +40,23 @@
 				"host": false,
 				"port": false
 			}
+		},
+		"mqtt": {
+			"host": "localhost",
+			"port": 8883,
+			"qos": 1,
+			"retain": false,
+			"secure": true,
+			"strictSSL": true,
+			"retries": 5,
+			"topic": {
+				"metric_topic": "server/metric/{accountid}/{deviceid}",
+				"health": "server/devices/{deviceid}/health",
+				"health_status": "device/{deviceid}/health",
+				"control_command": "device/{gwid}/control",
+				"ingestion_response": "device/{deviceid}/metric/response",
+				"actuation": "device/metric/{deviceid}/actuation"
```

Comment:
I don't think topics provided by the config are used anymore. In the SDK they have been moved to a non-configurable file because it was making the configuration file overly complicated for no real benefit.